src/modules/tabs/tabs.tsx

```diff
@@ -0,0 +1,66 @@
+// Modules
+import * as React from 'react';
+import {FlatList} from 'react-native';
+import Styled from 'styled-components/native';
+
+// Types
+import {IProps, IStyledText, IStyledTouchable, ITab} from './tabs.types';
```

Comment:
@ajaybushel I think we should follow this sub extension for interfaces as well. But let's discuss and sync up on the convention that should be followed across the board.